packages/rocketchat-livechat/client/views/app/livechatTriggers.js

```diff
@@ -72,7 +77,7 @@ Template.livechatTriggers.events({
 			closeOnConfirm: false,
 			html: false,
 		}, () => {
-			Meteor.call('livechat:removeTrigger', function(error, result) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["Meteor" is not defined.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2313371489/issues/source?bid=3199634&fileBranchId=3199636#l80)